pkg/endpointmanager/cell.go

```diff
@@ -238,9 +240,13 @@ func newDefaultEndpointManager(p endpointManagerParams) endpointManagerOut {
 type endpointSynchronizerParams struct {
 	cell.In
 
-	Clientset client.Clientset
+	Clientset      client.Clientset
+	CiliumEndpoint resource.Resource[*types.CiliumEndpoint]
```

Comment:
I think this will break CiliumEndpointSlices as all agents will be watching CEPs now, no?